### ISSUE TITLE
C# PID.Stop() fails with an "Unknown host" exception

### ISSUE THREAD

**Issue by r1sc** [AsynkronIT/gam#50]
Trying to call pid.Stop() fails on local actors with an "Unknown host" exception.
I managed to "fix" it by modifying TryAdd in ProcessRegistry.cs:

```csharp
public ValueTuple<PID, bool> TryAdd(string id, ActorRef aref)
{
	var pid = new PID()
	{
		Id = id,
		Ref = aref, //cache aref lookup
		Host = "nonhost" // local               <---- This fixes the error
	};
	var ok = _localActorRefs.TryAdd(pid.Id, aref);
	return ValueTuple.Create(pid, ok);
}
```

Maybe I am missing something here?

**Comment by rogeralsing** [comment]
Your fix is correct, the C# support is very much Alpha so this bug surfaced now that Remoting was added.
PR? ;)

**Comment by r1sc** [comment]
Fixed in #52
